modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3260,6 +3283,74 @@ void ONNXImporter::parseQMatMul(LayerParams& layerParams, const opencv_onnx::Nod
     addLayer(layerParams, node_proto);
 }
 
+// A * B + C = Y, we require that the dimension of A is [m, k], and the dimension of B is [n, k].
+// And the dim of output Y is [m, n]
+void ONNXImporter::parseQGemm(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
+{
+    int ninputs = node_proto.input_size();
+    CV_Assert(ninputs == 8 || ninputs == 9);
+
+    layerParams.type = "InnerProductInt8";
+
+    if (constBlobs.find(node_proto.input(3)) == constBlobs.end())
+        CV_Error(Error::StsNotImplemented, "Variable weights is not supported");
```

Comment:
`inp_sc`/`inp_zp` below can also be non-constant - so I think we should probably have new overload of `getGlob` that accepts error message and does this `CV_Error(Error::StSNotImplemented)` stuff under the hood.

Reply:
That's a good idea. In the near future, we do not have a plan to load this type ONNX op (like QGemm with non-constant `inp_sc/inp_zp`). I will try to do it.